inputfiles/addedTypes.json

```diff
@@ -19,6 +19,59 @@
                     ]
                 }
             },
+            "DocumentOrShadowRoot": {
+                "name": "DocumentOrShadowRoot",
+                "exposed": "Window",
```

Comment:
Probably the last request, but `exposed` is also redundant here for mixins, as by default it follows the host interface (which are `Document` and `ShadowRoot` in this case).